src/components/Todo/Topbar/filter-todo.component.jsx

```diff
@@ -1,15 +1,34 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { ALL, INCOMPLETE, COMPLETE } from "common/constants";
 
-const FilterTodo = () => {
+const FilterTodo = ({ handleFilterClick }) => {
```

Comment:
callback props should start with `on` prefix + event

```suggestion
const FilterTodo = ({ onFilter }) => {
```